tests/unittests/tests-bitfield/tests-bitfield.c

```diff
@@ -13,10 +13,12 @@
 
 #include "bitfield.h"
 
+#define BF_SIZE (5)
+
 static void test_bf_get_unset_empty(void)
 {
     int res = 0;
-    uint8_t field[5];
+    uint8_t field[BF_SIZE];
```

Comment:
Why not use the BITFIELD macro with `BF_SIZE == 40` here? That would make the code below a little bit easier to read:

```C
BITFIELD(field, BF_SIZE);
```